### DISCUSSION TITLE
How to make appcenter installing flatpak powered software system-wide by default?

### DISCUSSION THREAD

**Discussion by sauntor** [elementary/triage#293] [Q&A] [answered]
as the title of this Q&A, is it possible? the reason why try to do this, is that, some common apps like `Only Office `, `VLC`, etc.,  may be installed several times for each user, it waste hard disk space...

**Comment by lenemter** [comment, marked as answer]
You need to install flathub repository system-wide:

```
sudo flatpak remote-add --if-not-exists --system flathub https://flathub.org/repo/flathub.flatpakrepo
```

After that you can select system-wide install on the app page.

**Reply by sauntor** [reply]
Thanks! But what will happen if I just disable the filter of the predefined remote `freedesktop`  rather than adding a new `system` scoped `flathub`?

**Reply by lenemter** [reply]
I don't really know, but number of apps in freedesktop remote is exactly the same as in flathub. Guess you can just disable the filter.

```
lenemter@elementary-7:~$ flatpak remote-ls flathub --user | wc -l
3893
lenemter@elementary-7:~$ flatpak remote-ls freedesktop | wc -l
3893
```

**Reply by sauntor** [reply]
As this command shows:
```shell
flatpak remotes --columns=name,url
Name        URL
appcenter   https://flatpak.elementary.io/repo
freedesktop https://dl.flathub.org/repo/
````

The `system` remote `freedesktop` is exactly the `flathub` with some apps filtered out.

I disabled its filter already, and it seems no strange things will happen.
